app/helpers.php

```diff
@@ -209,20 +209,18 @@ function lazy_load_image($url, $class = '', $alt = '')
 
 function nav_links()
 {
-    $links = [];
+    $links = landing_nav_links();
```

Comment:
So... have the same order as in the current prod, but the same subset as in flyte's design?